scripts/sync-dashboards-and-alert-rules.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
This script need to be made executable and committed (git tracks executable bit for files):

```
chmod +x scripts/sync-dashboards-and-alert-rules.sh
git add scripts/sync-dashboards-and-alert-rules.sh
```